src/clinicaltrialgov.ts

```diff
@@ -65,10 +83,40 @@ export function findNCTNumber(study: ResearchStudy): string | null {
 export function parseClinicalTrialXML(fileContents: string): Promise<ClinicalStudy> {
   const parser = new xml2js.Parser();
   return parser.parseStringPromise(fileContents).then((result) => {
-    return (result as TrialBackup).clinical_study;
+    if (isTrialBackup(result)) {
+      return result.clinical_study;
+    } else {
+      throw new Error('Unable to parse trial as valid clinical study XML');
+    }
   });
 }
 
+const CLINICAL_STATUS_MAP = new Map<StatusEnum, ResearchStudyStatus>([
+  ['Active, not recruiting', 'closed-to-accrual'],
+  ['Completed', 'completed'],
+  // FIXME: This does not appear to have a proper mapping
+  ['Enrolling by invitation', 'active'],
+  ['Not yet recruiting', 'approved'],
+  ['Recruiting', 'active'],
+  ['Suspended', 'temporarily-closed-to-accrual'],
+  ['Terminated', 'administratively-completed'],
+  ['Withdrawn', 'withdrawn'],
+  // FIXME: Guessing on this mapping
+  ['Available', 'active'],
+  ['No longer available', 'closed-to-accrual'],
+  ['Temporarily not available', 'temporarily-closed-to-accrual'],
+  // FIXME: what?
+  ['Approved for marketing', 'active'],
+  // FIXME: This does not appear to have a proper mapping
+  ['Withheld', 'in-review'],
+  // FIXME: This does not appear to have a proper mapping
+  ['Unknown status', 'in-review']
```

Comment:
This one is a tough one, I think I would rather it default to 'active' and have the user find out after the fact that it may or may not be in that state.

Reply:
This is going to stay 'in-review' so that it isn't falsely closing the trial, but will also allow the user to find out more information about the trial.